### DISCUSSION TITLE
Inquiry Regarding Edge Extensions Review During Christmas Period

### DISCUSSION THREAD

**Discussion by grayfox333** [microsoft/MicrosoftEdge-Extensions#124] [General]
Hello,
I would like to know in advance whether the Edge Extensions review team has any plans for vacation during the Christmas period, so that we can make informed decisions in planning our development schedule.
Thank you!
Raj Jiang

**Comment by Bhuvana-Priya** [comment]
Hi,

We apologize for the delay.

We would like to let you know that there would be no vacations when it comes to review times for Edge Extensions. The reviews would follow the standard time regardless.

Thanks,
Bhuvana
